package.json

```diff
@@ -13,6 +13,7 @@
     "start:dev": "./node_modules/.bin/webpack --watch --env.NODE_ENV=development --progress --colors --mode=development --debug --devtool cheap-eval-source-map --output-pathinfo",
     "start:microservices": "docker-compose up -d ghopper ghopper-walking live-tracking map",
     "start:node": "docker-compose up node",
+    "start:backend": "npm run build:dev; npm run start:microservices; npm run start:node",
```

Comment:
you only have to run the first two commands one time, and it should hot reload & the microservices persist in the background, so i don't think we need this